pages/phases/getting-started/measuring-a-hiring-action.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: article
+permalink: /hiring-phases/getting-started/collecting-data/
+section: hiring-phases
+title: Collecting data
+sidenav: hiring-phases
+phase: getting-started
+sub-phase-order: 030
+---
+
+## Collecting baseline data
+
+<p class="usa-intro">
+  To gauge the success of your hiring action, collect baseline data about recent delegated examination (DE) hiring actions at your agency. You'll compare this information with the results you receive when your new hiring action is complete.
+</p>
+
+For one to two recent DE actions, gather:
+
+1. **Current qualified rate**: How many applications were considered qualified by an HR specialist.
+2. **Current selection rate**: The number of selections made off of the resulting certificate, represented numerically and as a percentage of qualified applications.
+
+## Measuring your process
+
+<p class="usa-intro">
+  To gauge the success of the hiring process, measure applicant flow through the process and confidence in the process.
+</p>
+
+1. **Hiring funnel**: How many applicants progress through each phase: Resume review, interview 1, and interview 2.
+2. **Level of burden**: Have internal resources track their time: SMEs, HR, hiring manager, schedulers, project managers.
+
+## Measuring your results
+
+<p class="usa-intro">
+  To gauge the success of your hiring action, measure applicant quality and the applicant pool.
+</p>
+
+1. **Selections**: How many applicants make it onto certificates by category and how many applicants are ultimately selected by a hiring manager.
+2. **Adverse impact**: Track demographics of the applicant pool after each phase. Demographics include race and gender. Also, track eligibility data such as federal employees, Veterans, etc.
```

Comment:
we need to add more guidance on how to identify fed vs contractor vs private. Maybe we can have applicants submit this information voluntarily instead of guessing from their resume?

Reply:
We could ask the USAJOBS program office to add it. They'll have to go through PRA. So it won't be there in time for our next round of pilots, but maybe in future.

Reply:
i guess maybe we can add this as an HR responsibility for phase 2...

Reply:
We're taking a 2 pronged attack:
1. Reach out to Michelle with text to change the usajobs profile question.
2. We'll do this for the next round of the pilots.